admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -765,7 +771,7 @@ def _makeIndex(self, database, table):
         self._log.info('Generating index %r.%r', self.indexDb, metaTable)
 
         # try to delete existing table first
-        self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
+        #self.czarWmgr.dropTable(self.indexDb, metaTable, mustExist=False)
```

Comment:
It turns out the data loader script already has this option:

-e, --delete-tables   If specified then existing tables in database will be
                        deleted if they exist, this includes both data and
                        metadata.

Should I just rely on it here? I can propagate this flag into the code which makes the secondary index.